File: src/test/java/com/roscopeco/jasm/model/IfNullNonNullTest.java
package com.roscopeco.jasm.model;

public interface IfNullNonNullTest {
    boolean testIfNullWhenNullPasses();
    boolean testIfNullWhenNonNullPasses();
    boolean testIfNonNullWhenNullPasses();
    boolean testIfNonNullWhenNonNullPasses();
}

File: src/test/kotlin/com/roscopeco/jasm/model/IfNullNonNullTest.kt
package com.roscopeco.jasm.model

interface IfNullNonNullTest {
    fun testIfNullWhenNullPasses(): Boolean
    fun testIfNullWhenNonNullPasses(): Boolean
    fun testIfNonNullWhenNullPasses(): Boolean
    fun testIfNonNullWhenNonNullPasses(): Boolean
}